packages/react-ui/src/theme/carto-theme.js

```diff
@@ -1175,16 +1175,10 @@ export const cartoThemeOptions = {
   }
 };
 
-export function createTheme(options = {}) {
```

Comment:
Are we sure we can remove default options?

Reply:
As described in `documentation` PR, the way to overwrite `cartoThemeOptions` is equal for both implementations in this way.

Also, passing to `createTheme` and object like:
```javascript
const mainColor = { palette: { primary: { main: '#80000' }}}
```
results in an error. Do you have any other idea to handle this?

Reply:
What if we leave the optional param, and just expose the method?

That way we could also allow overriding certain properties, so we would eventually get 3 options for the user:

- call to createTheme without params (one line from the client)
- call to createTheme with certain options (maybe just colors)
- completely custom case, using just cartoThemeOptions (what is currently being used from template)

/cc @borja-munoz